test/saithrift/src/switch_sai_rpc_server.cpp

```diff
@@ -2386,25 +2386,28 @@ class switch_sai_rpcHandler : virtual public switch_sai_rpcIf {
                   attr_list[i].value.u8 = attribute.value.u8;
                   break;
               case SAI_MIRROR_SESSION_ATTR_VLAN_TPID:
-                  attr_list[i].value.u16 = attribute.value.u16;
```

Comment:
u16 on client is implemented using i16, which is actually s16. Please see the comment in switch.py line 55 for reference.